### PR TITLE
Main branch should take over master branch 

### LINKED ISSUE #235: Main/Master branch Transition
## Expected Behavior

Only one branch should be the the main branch. 

## Actual/Current Behavior

At the moment, some developers are defaulting to the master branch. I didn't catch that in the PR process until now when looking at the git tree. We should only support the main branch and remove the master brach from the repo entirely to reduce confusion.

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
_List Changes Introduced by this PR_
1. Keep main and master up to date with each other.
2. Remove master branch afterwords. 

## Purpose
The master branch has now been deprecated on Github and should now be attached to main. We should only be supporting one of the other to reduce confusion within any repo. 

## Approach
Merge all changes from master to the main branch. Afterwards, I will delete the master branch.

## Pre-Testing TODOs
- [ ] Merge master with main.
- [ ] Delete!

## Testing Steps
None except following through. 

## Learning
This is in relation to the change introduced here #206

Closes #235

### CODE REVIEW

File: standards/qa-process.md [hunk 1]
```diff
@@ -47,33 +48,36 @@
...
 3. If applicable, review for each user types abilities and limitations.
 4. Credentials are given to the team.
 5. If QA is billable to the client, such as for T&M projects, PM will add the QA team to the Tick project.
 
 # Testing Process
 ## For QA:
 
-1. QA Testers will create a “Working” issue to work from, copy the Acceptance Test from the “QA Sprint” issue into the first comment, and use that to check off each tested feature.
-2. Discussions should occur constantly within the “Working” issue, tagging people as necessary.
+1. QA will set up a project board for this sprint by copying from the example board on github (link).
```
Comment (kmausolf) [inline standards/qa-process.md:63]: Is there meant to be a link here?  Or will it be added later?

File: standards/qa-process.md [hunk 2]
```diff
@@ -111,26 +116,42 @@ a. Labels:
...
 4. If the test is successful, QA Tester adds the “Testing Passed” label to the issue. Issue assigned to POC.
 5. If the testing failed, QA will add feedback on the issue and add the “Testing Failed” label.
-6. Upon completion of the QA Sprint, the POC will close the “Working” issues and “QA Sprint” issue.
+6. If the issue is marked as “Testing Passed”, drag the item to “Done” and close the issue if applicable
+7. Upon completion of the QA Sprint, the POC will close the “Working” issues and “QA Sprint” issue.
+
+## Post Sprint Meeting
+This is an optional meeting for the PM and will be scheduled on the proposed end date with the lead dev if testing is not completed to the satisfaction of definition of done.
+1. QA and dev will review the acceptance test to see what items may have not been marked off.
+2. QA and dev will go over any issues that are still open and issues that need a more thurough explanation if neccessary.
```
Comment (kmausolf) [inline standards/qa-process.md:128]: Typo here:  thorough

**Review by kmausolf — COMMENTED** [review]
Great work on the QA doc update, glad we're beefing up the QA process!  I'm not sure what this has to do with the `main` branch update, though.  Is this a mis-titled PR?

**Comment by michaelachrisco** [comment]
> Great work on the QA doc update, glad we're beefing up the QA process! I'm not sure what this has to do with the `main` branch update, though. Is this a mis-titled PR?

Thanks for taking a look! I am not the one that added any of these changes, rather I am just puling these into the correct branch (Main). These changes were accidentally merged into master and NOT main (witch is now the default branch in the repo). Does that make sense?

File: standards/qa-process.md (hunk 2 above)
Comment (michaelachrisco) [reply standards/qa-process.md:128]: Ill throw this into another Issue/PR. Thanks!

**Comment by kmausolf** [comment]
> > Great work on the QA doc update, glad we're beefing up the QA process! I'm not sure what this has to do with the `main` branch update, though. Is this a mis-titled PR?
> 
> Thanks for taking a look! I am not the one that added any of these changes, rather I am just puling these into the correct branch (Main). These changes were accidentally merged into master and NOT main (witch is now the default branch in the repo). Does that make sense?

Ohhhh!  Makes sense!

**Review by kmausolf — APPROVED** [review]

**Review by suchprogramming — APPROVED** [review]
Thanks for doing such a good job with our S&P repo Michael!

File: standards/qa-process.md (hunk 1 above)
Comment (michaelachrisco) [reply standards/qa-process.md:63]: Good question, we should probably look into that!

**Comment by michaelachrisco** [comment]
Ill hold off til tomorrow (at least) on this one just in case anyone want to change their `master` references on their repos. Thanks for the reviews!

**Pull request merged**